src/components/TopBar/TopBar.vue

```diff
@@ -339,20 +339,20 @@ export default {
 	}
 
 	.talk-sidebar-callview & {
-		margin-right: var(--default-clickable-area);
+		margin-inline-end: var(--default-clickable-area);
```

Comment:
wouldd also add here `align-items: flex-start;` to avoid:
b | a
-- | --
<img width="290" alt="image" src="https://github.com/user-attachments/assets/4dd407c0-d350-45ea-aa97-82fd23f7df0d" /> | <img width="288" alt="image" src="https://github.com/user-attachments/assets/3e24c1b5-df9c-4165-ae52-643060f53b83" />